system_baseline/models.py

```diff
@@ -78,10 +78,10 @@ def validate_existing_system(self, system_id):
                     "System {} already associated with this baseline".format(system_id)
                 )
 
-    def add_mapped_system(self, system_id):
+    def add_mapped_system(self, system_id, groups=None):
         self.validate_existing_system(system_id)
         new_mapped_system = SystemBaselineMappedSystem(
-            system_id=system_id, account=self.account, org_id=self.org_id
+            system_id=system_id, account=self.account, org_id=self.org_id, groups=groups
```

Comment:
How do we update groups on a SystemBaselineMappedSystem table?

Reply:
Never mind. Found it, ... updating within HSP by calling baselines internally https://issues.redhat.com/browse/ESSNTL-5248